sql-plugin/src/main/java/com/nvidia/spark/rapids/RapidsHostColumnVector.java

```diff
@@ -143,7 +150,27 @@ public ColumnarArray getArray(int rowId) {
 
   @Override
   public ColumnarMap getMap(int ordinal) {
-    throw new IllegalStateException("Maps are currently not supported by rapids cudf");
+    ai.rapids.cudf.ColumnViewAccess<HostMemoryBuffer> structHcv = cudfCv.getChildColumnViewAccess(0);
```

Comment:
So why are we not just casing the ColumnViewAccess to a ColumnViewCore?  Eventually cudf should be update to return that type directly instead.

Reply:
Sure I can do that.

Reply:
Done.